.coveragerc

```diff
@@ -0,0 +1,29 @@
+[html]
+show_contexts = true
+skip_covered = false
+
+[paths]
+source =
+  mysite
+
+[report]
+skip_covered = true
+show_missing = true
+exclude_lines =
+  \#.*no cover
+  \s*raise AssertionError\b
+  \s*raise NotImplementedError\b
+  \s*return NotImplemented\b
+  \s*raise$
+  if __name__ == ['"]__main__['"]:$
+
+[run]
+branch = true
+cover_pylib = false
+dynamic_context = test_function
+parallel = true
+relative_files = true
+source =
+  tests
```

Comment:
You don't have this top-level folder so it's probably unnecessary to use this setting. OTOH, it may be nice to move the tests to the repository root so perhaps keeping it as it is is warranted.

Reply:
Okay, I will create the directory in the next PR.

Reply:
Marking this as resolved.